pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -3136,6 +3392,10 @@ in
     };
   };
 
+  zenbones-nvim = super.zenbones-nvim.overrideAttrs {
+    dependencies = with self; [ lush-nvim ];
```

Comment:
is lush a real dependency or was it used to generate the colorschme ? I dont know how it works.

Reply:
Might actually be optional with configuration https://github.com/zenbones-theme/zenbones.nvim/tree/main?tab=readme-ov-file#installation